test/unit/audit_mailer_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class AuditMailerTest <ActionMailer::TestCase
+  def setup
+    disable_orchestration
+    SETTINGS[:entries_per_page] = 10
+    # Add 'update' action to audits
+    @audit = FactoryGirl.create(:audit)
+    as_admin do
+      # Add 'create' action to audits
+      @host = FactoryGirl.create(:host)
+    end
+    User.current = users :admin
```

Comment:
Why use as_admin in a block above (which sets User.current) and still leave User.current set here?  Is this line needed or can we delete it?  It's best to avoid setting User.current unilaterally as it can lead to bugs when it's being set in tests but not in real life.